Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py

```diff
@@ -0,0 +1,935 @@
+            readable_output=f"## No, user - {args.get('userName')} is not a member of group - {args.get('groupName')}.",
+            outputs_prefix='STA.EXIST.USER.GROUP',
+            outputs=response
+        )
+
+
+def add_user_group_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-add-user-group command. Add user to a specific group. """
+
+    response = client.add_user_group_sta(userName=args.get('userName'), groupName=args.get('groupName'))
+    if not response:
+        return CommandResults(
+            readable_output=NO_RESULT_MSG,
+        )
+
+    return CommandResults(
+        readable_output=f"## User - {args.get('userName')} successfully added to the group - {args.get('groupName')}.",
+        outputs_prefix='STA.UPDATE.USER.GROUP',
+        outputs_key_field=['user_id', 'group_id'],
+        outputs=response
+    )
+
+
+def remove_user_group_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-remove-user-group command. Remove user from a specific group. """
+
+    response = client.remove_user_group_sta(userName=args.get('userName'), groupName=args.get('groupName'))
+    if not response:
+        return CommandResults(
+            readable_output=NO_RESULT_MSG,
+        )
+
+    return CommandResults(
+        readable_output=f"## User - {args.get('userName')} successfully removed from the group - {args.get('groupName')}.",
+        outputs_prefix='STA.UPDATE.USER.GROUP',
+        outputs_key_field=['user_id', 'group_id'],
+        outputs=response
+    )
+
+
+def get_logs_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-get-logs command. Get user's logs. """
+
+    response = client.get_logs_sta(userName=args.get('userName'), since=args.get('since'),
+                                   until=args.get('until'), limit=args.get('limit'))
+    if not response:
+        return CommandResults(
+            readable_output=NO_RESULT_MSG,
+        )
+    header_sequence = ['timeStamp', 'userName', 'logType', 'credentialType', 'actionText', 'resultText', 'message',
+                       'applicationName', 'policyName', 'state', 'operationType', 'operationObjectType',
+                       'operationObjectName', 'serial', 'ip']
+
+    return CommandResults(
+        readable_output=tableToMarkdown("Logs : ", response, headers=header_sequence,
+                                        headerTransform=pascalToSpace, removeNull=True),
+        outputs_prefix='STA.LOGS',
+        outputs=response
+    )
+
+
+def validate_tenant_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-validate-tenant command. Validate key and permissions. """
+
+    client.validate_tenant_sta()
+    return CommandResults(
+        readable_output="## The requested tenant is accessible.",
+        outputs_prefix='STA.VALIDATE.TENANT',
+        outputs=True
```

Comment:
If you dont need this data for further use, so I dont need to create a context data, you can just return a human readable

Reply:
Done. Removed outputs and outputs_prefix field from **sta-validate-tenant** command.

Reply:
@tallieber 
We are using this context data in the playbook. So no need to remove the same.